spec/index.html

```diff
@@ -266,35 +266,18 @@ <h2>A Canonical form of N-Quads</h2>
       MUST NOT use the datatype IRI part of the <a href="#grammar-production-literal">literal</a>,
       and are represented using only <a href="#grammar-production-STRING_LITERAL_QUOTE">STRING_LITERAL_QUOTE</a>.
     </li>
-    <!--li><code><a href="#grammar-production-HEX">HEX</a></code> MUST use only uppercase letters (<code>[A-F]</code>).</li-->
-    <li>Characters MUST NOT be represented by <code><a href="#grammar-production-UCHAR">UCHAR</a></code>.</li>
+    <li><code><a href="#grammar-production-HEX">HEX</a></code> MUST use only uppercase letters (<code>[A-F]</code>).</li>
     <li>Within <a href="#grammar-production-STRING_LITERAL_QUOTE">STRING_LITERAL_QUOTE</a>,
       the characters 
       <code>U+0022</code>, <code>U+005C</code>, <code>U+000A</code>, <code>U+000D</code>
-       MUST be encoded using <code><a href="#grammar-production-ECHAR">ECHAR</a></code>.
-      <code><a href="#grammar-production-ECHAR">ECHAR</a></code> MUST NOT be used for characters that are
-      allowed directly in
-      <code><a href="#grammar-production-STRING_LITERAL_QUOTE">STRING_LITERAL_QUOTE</a></code>. </li>
+      MUST be encoded using <code><a href="#grammar-production-ECHAR">ECHAR</a></code>.
+      Characters in the range from <code>U+0000</code> to <code>U+001F</code> and <code>U+007F</code>
+      not represented using <code><a href="#grammar-production-ECHAR">ECHAR</a></code>
```

Comment:
```suggestion
      Characters in the range from <code>U+0000</code> to <code>U+001F</code> and <code>U+007F</code>
      that are not represented using <code><a href="#grammar-production-ECHAR">ECHAR</a></code>
```

Reply:
Digression from this PR but reading the section, the text is a bit vague and unsure of itself.

> which has less variability in layout.

Not less - it has no variability in layout, except order of quads/triples.

> NOTE
> A canonical form of N-Quads can be used to ensure that variations in the syntactic representation of terms within that quad are determined; each code point can be represented by only one of [UCHAR](https://pr-preview.s3.amazonaws.com/w3c/rdf-n-quads/pull/27.html#grammar-production-UCHAR), [ECHAR](https://pr-preview.s3.amazonaws.com/w3c/rdf-n-quads/pull/27.html#grammar-production-ECHAR), or unencoded character, where the relevant production allows for a choice in representation.

I'm not sure what it is saying -- "variations" are what we are trying to avoid.

A more direct statement (which can be further refined)

> The N-Quads syntax allows choices for the representation and layout of RDF data. _The_ canonical form of N-Quads provides a unique syntactic representation of any triple or quad.

May be call out the fact that one triple/quad is one line and up to order, the representation is unique.

It can be a plain paragraph.